spec/sh-options.test.sh

```diff
@@ -318,16 +330,8 @@ echo $?
 set +x
 echo $SHELLOPTS | grep -q xtrace
 echo $?
-## STDOUT:
-1
-0
-1
```

Comment:
Unfortunately it looks I made a change while you were moving repos, and this file got "clobbered"

Please revert this change -- hopefully there is an easy way to do it

Reply:
ok done